### DISCUSSION TITLE
EDA-GPT and data discovery

### DISCUSSION THREAD

**Discussion by Extremys** [RamiAwar/dataline#273] [Ideas]
I've heard about this project recently, there is an interesting approach around data discovery that could complete Dataline project approach, what do you think?
https://github.com/shaunthecomputerscientist/EDA-GPT

**Comment by RamiAwar** [comment]
Yeah I saw it a few days ago as well, it looks cool!

I do see more explorative data analysis features in DataLine in the future. Just not sure about when we'd run them yet since we deal with potentially massive DBs which means costly queries (maybe, we could check first).

What features did you think about having in DataLine? @Extremys
